README.md

```diff
@@ -404,13 +404,29 @@ via:
 reek -c whatever/.todo.reek lib/
 ```
 
-Note that if you re-run
+It's important to understand that the number one use case of the `--todo` flag
+is to be run once at the beginning of the introduction of Reek to ease the transition.
+If you find yourself re-running Reek with the `--todo` flag in order to silence new warnings
+you're defeating the purpose of both the `--todo` flag and of Reek itself.
+
+As a consequence, this
```

Comment:
I would remove the full invocation below, and put something like 'running Reek with the `--todo` flag again'.